src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -70,7 +77,11 @@ public EventTypeController(final EventTypeRepository eventTypeRepository,
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<?> createEventType(@Valid @RequestBody final EventType eventType,
                                              final Errors errors,
-                                             final NativeWebRequest nativeWebRequest) {
+                                             final NativeWebRequest nativeWebRequest)
+    {
+        if (featureToggleService.isFeatureEnabled("disable_event_type_creation")) {
```

Comment:
I see that it got merged with this comment not been addressed. Even if it's a very simple matter I don't feel comfortable with this approach. We should try to reply or address every comment.